### DISCUSSION TITLE
Issue while runnig the model

### DISCUSSION THREAD

**Discussion by NS-Nik** [NOAA-EMC/WW3#1214] [Q&A]
Hi! 

I am running the model using the mpirun and I get the following repeating mistake: 

<img width="953" alt="Снимок экрана 2024-04-09 в 15 01 03" src="https://github.com/NOAA-EMC/WW3/assets/104583032/8aeeb9ae-357b-402e-a1c0-2c0c1220965b">

It pops up  in the terminal window with an interval of about an hour, while model still continues the run, but at some point the calculation freezes. 

It seems that this is the problem with mpi.  

Maybe you have some ideas how to fix it or at least how to collect more information about this mistake.

**Comment by mickaelaccensi** [comment]
it seems to be related to your version of openmpi : 
https://github.com/open-mpi/ompi/issues/9218

Best,

Mickael
